snuba/consumers/consumer_builder.py

```diff
@@ -86,6 +86,9 @@ def __init__(
         self.storage = get_writable_storage(storage_key)
         self.bootstrap_servers = kafka_params.bootstrap_servers
         self.consumer_group = kafka_params.group_id
+        self.replacements_producer = None
+        self.commit_log_producer = None
```

Comment:
A bit of a nitpick here but I would probably set this in the else statement where `replacements_topic` and `commit_log_topic` get set to None just to mirror how those variables get set

Reply:
Sure, that makes sense to me